app/controllers/subject_viewer.coffee

```diff
@@ -148,15 +154,31 @@ class SubjectViewer extends Controller
     @activate selectedIndex
 
   onClickSatellite: ->
+    if not @satelliteImage.hasClass 'active'
+      AnalyticsLogger.logEvent User.current?.zooniverse_id,@classification.subject.zooniverseId,'map',''
     @satelliteImage.add(@satelliteToggle).toggleClass 'active'
 
   onClickSignIn: ->
     $(window).trigger 'request-login-dialog'
 
   onClickFavorite: ->
+    AnalyticsLogger.logEvent User.current?.zooniverse_id,@classification.subject.zooniverseId,'favorite',@classification.id
     @classification.updateAttribute 'favorite', true
 
+  onClickFacebook: ->
+    AnalyticsLogger.logEvent User.current?.zooniverse_id,@classification.subject.zooniverseId,'facebook',@classification.id
+
+  onClickTweet: ->
+    AnalyticsLogger.logEvent User.current?.zooniverse_id,@classification.subject.zooniverseId,'tweet',@classification.id
+
+  onClickPin: ->
+    AnalyticsLogger.logEvent User.current?.zooniverse_id,@classification.subject.zooniverseId,'pin',@classification.id
+
+  onClickTalk: ->
+    AnalyticsLogger.logEvent User.current?.zooniverse_id,@classification.subject.zooniverseId,'talk',@classification.id
+
```

Comment:
Petty, but why split up the `onClickFavorite` and `onClickUnfavorite` methods like this? They kinda belong together.